src/test/resources/data/fns_pointseries.json

```diff
@@ -1,466 +1,484 @@
 {
-  "batch": [
+  "batch":[
     {
-      "args": {
-        "x": "@timestamp",
-        "y": "final"
+      "functionName":"pointseries",
+      "args":{
+        "x":"@timestamp",
+        "y":"final"
       },
-      "context": {
-        "columns": [
+      "context":{
+        "type":"datatable",
+        "columns":[
           {
-            "name": "@timestamp",
-            "type": "date"
+            "id":"@timestamp",
+            "name":"@timestamp",
+            "meta":{
+              "type":"date"
+            }
           },
           {
-            "name": "value",
-            "type": "number"
+            "id":"value",
+            "name":"value",
+            "meta":{
+              "type":"number"
+            }
           },
           {
-            "name": "label",
-            "type": "string"
+            "id":"label",
+            "name":"label",
+            "meta":{
+              "type":"string"
+            }
           },
           {
-            "name": "mean",
-            "type": "number"
+            "id":"mean",
+            "name":"mean",
+            "meta":{
+              "type":"number"
+            }
           },
           {
-            "name": "rnd",
-            "type": "number"
+            "id":"rnd",
+            "name":"rnd",
+            "meta":{
+              "type":"number"
+            }
           },
           {
-            "name": "final",
-            "type": "number"
+            "id":"final",
+            "name":"final",
+            "meta":{
+              "type":"number"
+            }
           }
         ],
-        "rows": [
+        "rows":[
           [
-            1565474400000,
+            1586642400000,
             0,
             "q:* > sum(taxless_total_price)",
-            2574,
-            1.04,
-            2676.96
+            2423,
+            0.9,
+            2180.7
           ],
```

Comment:
Is it fine to hardcode these data in payload json or it should be modified based on current time/etc?